cfgov/ask_cfpb/models/django.py

```diff
@@ -127,8 +123,7 @@ def __str__(self):
         return self.name
 
     def featured_answers(self, language):
-        from .pages import AnswerPage
```

Comment:
I know this functionality will go away eventually with removal of the landing page, but I'm wondering if in the interim we should move this over to the `AnswerPage`?

Reply:
I don't see an advantage here. We can redo tests and migrations, etc., but we'll end up taking it back out in the end. I don't see the value for a short-term condition.